docs/docs/porting-part-of-a-html-site-to-gatsby.md

```diff
@@ -521,11 +521,48 @@ website-domain
 {/* highlight-end */}
 ```
 
+## Porting more parts
+
+To replace multiple parts of a website with a single Gatsby application there are a few additional steps:
+
+1. Mirror the website structure within the Gatsby `/src/pages` folder.
+
+2. Adjust the `pathPrefix` Gatsby configuration option to reflect the new root path at which the Gatsby application will be served.
+
+3. Ensure the `to` props on all Gatsby `<Link>` components are correct relative to `/src/pages/` as the application root.
```

Comment:
general question (maybe adressed also early on this page): here we use `<Link>` for all pages in the old and new website or only for the new website (preloading is compatible with old html pages?)

Reply:
i think you adressed this in the next point 3

Reply:
The [docs](https://www.gatsbyjs.org/docs/gatsby-link/#replace-a-tags-with-the-link-tag-for-local-links) warn against using `<Link>` for external sites, I took that to mean anything outside the Gatsby app but looking back it may just be external to the website as a whole.

Once `pathPrefix` is not being used in a project, I don't know if there is a reason not to use `<Link>` for non Gatsby, internal pages. That is a possible small clarification to add to the [Link page](https://www.gatsbyjs.org/docs/gatsby-link/#replace-a-tags-with-the-link-tag-for-local-links). Will dig a little to find the answer.